# Norwegian Nynorsk translation for update-manager
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the update-manager package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2011-10-18 03:11+0000\n"
"PO-Revision-Date: 2011-05-11 19:04+0000\n"
"Last-Translator: Tor Egil Hoftun Kvæstad <toregilhk@hotmail.com>\n"
"Language-Team: Norwegian Nynorsk <nn@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2011-10-18 14:53+0000\n"
"X-Generator: Launchpad (build 14157)\n"
"X-Language: nn_NO\n"
"X-Source-Language: C\n"

#. TRANSLATORS: download size is 0
#: ../DistUpgrade/utils.py:416 ../UpdateManager/Core/utils.py:416
msgid "0 kB"
msgstr ""

#. TRANSLATORS: download size of very small updates
#: ../DistUpgrade/utils.py:419 ../UpdateManager/Core/utils.py:419
msgid "1 kB"
msgstr ""

#. TRANSLATORS: download size of small updates, e.g. "250 kB"
#: ../DistUpgrade/utils.py:422 ../UpdateManager/Core/utils.py:422
#, python-format
msgid "%.0f kB"
msgstr ""

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:425 ../UpdateManager/Core/utils.py:425
#, python-format
msgid "%.1f MB"
msgstr "%.1f MiB"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:210 ../DistUpgrade/distro.py:438
#, python-format
msgid "Server for %s"
msgstr "Tenar for %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:228 ../DistUpgrade/distro.py:234
#: ../DistUpgrade/distro.py:250
msgid "Main server"
msgstr "Hovudtenar"

#: ../DistUpgrade/distro.py:254
msgid "Custom servers"
msgstr "Eigendefinerte tenarar"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:141
msgid "Could not calculate sources.list entry"
msgstr "Klarte ikkje å rekna ut oppføringa i sources.list"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:244
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Klarte ikkje å finna nokon pakkefiler. Kanskje dette ikkje er ein Ubuntu-"
"disk eller rett arkitektur?"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:287
msgid "Failed to add the CD"
msgstr "Klarte ikkje å leggja til CD-en"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:288
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Det oppstod ein feil under lesing av CD-en, oppgraderinga vert avbroten. "
"Dette lyt rapporterast som ein feil, dersom dette er ein gyldig Ubuntu-CD.\n"
"\n"
"Feilmeldinga var:\n"
"\"%s\""

#: ../DistUpgrade/DistUpgradeCache.py:138
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Fjern dårleg pakke"
msgstr[1] "Fjern dårlege pakkar"

#: ../DistUpgrade/DistUpgradeCache.py:141
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Pakken '%s' er øydelagd, og lyt leggjast inn på nytt, men ingen av dei "
"nødvendige arkiva vart funne. Vil du fjerna pakken no og halda fram?"
msgstr[1] ""
"Pakkene '%s' er ødelagde, og lyt leggjast inn på nytt, men ingen av dei "
"nødvendige arkiva vart funne. Vil du fjerna pakkene no og halda fram?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:239
msgid "The server may be overloaded"
msgstr "Tenaren kan vera oppteken"

#: ../DistUpgrade/DistUpgradeCache.py:352
msgid "Broken packages"
msgstr "Skadde pakkar"

#: ../DistUpgrade/DistUpgradeCache.py:353
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Systemet ditt inneheld øydelagde pakkar som ikkje kunne reparerast med denne "
"programvara. Reparer dei med synaptic eller apt-get før du held fram."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:648
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"Eit problem som ikkje kan løysast automatisk oppstod medan oppgraderinga "
"vart førebudd:\n"
"%s\n"
"\n"
" Dette kan vera årsakene:\n"
" * Oppgradering til ei tidleg utgåve (før-utgåva) av ein Ubuntu-versjon\n"
" * Bruken av den noverande før-utgavå av ein Ubuntu-versjon\n"
" * Uoffisielle programpakkar som ikkje kjem frå Ubuntu\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:658
msgid "This is most likely a transient problem, please try again later."
msgstr "Dette er truleg berre eit kortvarig problem. Prøv om att seinare."

#: ../DistUpgrade/DistUpgradeCache.py:661
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug update-manager' in a terminal."
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:666
#: ../UpdateManager/UpdateManager.py:1023
msgid "Could not calculate the upgrade"
msgstr "Klarte ikkje å førebu oppgraderinga"

#: ../DistUpgrade/DistUpgradeCache.py:717
msgid "Error authenticating some packages"
msgstr "Klarte ikkje å stadfesta identiteten åt somme pakkar"

#: ../DistUpgrade/DistUpgradeCache.py:718
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Somme pakkar kunne ikkje verta godkjende. Dette kan vera eit kortvarig "
"nettverksproblem, så du bør prøve om att seinare. Sjå under for lista over "
"pakkar som ikkje kunne godkjennast."

#: ../DistUpgrade/DistUpgradeCache.py:738
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"Pakken '%s' er merkt for fjerning, men er i svartelista for fjerning."

#: ../DistUpgrade/DistUpgradeCache.py:742
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Den nødvendige pakken «%s» er merkt for fjerning."

#: ../DistUpgrade/DistUpgradeCache.py:751
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Freistar å installera den svartelista versjon '%s'"

#: ../DistUpgrade/DistUpgradeCache.py:869
#, python-format
msgid "Can't install '%s'"
msgstr "Kan ikkje installera '%s'"

#: ../DistUpgrade/DistUpgradeCache.py:870
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug update-manager' in a terminal."
msgstr ""

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:881
msgid "Can't guess meta-package"
msgstr "Klarte ikkje å gjetta på meta-pakke"

#: ../DistUpgrade/DistUpgradeCache.py:882
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Systemet ditt inneheld ingen av pakkane ubuntu-desktop, kubuntu-desktop "
"eller edubuntu-desktop og det var ikkje mogeleg å finna ut kva ubuntu-"
"versjon du brukar.\n"
"Installer ein av desse pakkane ved å bruka synaptic eller apt-get før du "
"fortset."

#: ../DistUpgrade/DistUpgradeController.py:100
msgid "Reading cache"
msgstr "Les mellomlager"

#: ../DistUpgrade/DistUpgradeController.py:205
msgid "Unable to get exclusive lock"
msgstr "Klarte ikkje å få einerett"

#: ../DistUpgrade/DistUpgradeController.py:206
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Dette tyder vanlegvis på at eit anna pakkehandsamingsprogram (slik som apt-"
"get eller aptitude) køyrer. Avslutt det programmet først."

#: ../DistUpgrade/DistUpgradeController.py:239
msgid "Upgrading over remote connection not supported"
msgstr "Å oppgradera over fjerntilkobling er ikkje støtta"

#: ../DistUpgrade/DistUpgradeController.py:240
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Du køyrer oppgraderinga over ei ssh-tilkopling med grensesnitt som ikkje "
"støttar dette. Freist ei oppgradering i tekstmodus med 'do-release-"
"upgrade'.\n"
"\n"
"Oppgraderinga vil no avbrytast. Prøv utan ssh."

#: ../DistUpgrade/DistUpgradeController.py:254
msgid "Continue running under SSH?"
msgstr "Forsett køyringa under SSH?"

#: ../DistUpgrade/DistUpgradeController.py:255
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Denne økta køyrer over ssh. Det er ikkje tilrådd å oppgradera på denne "
"måten, sia systemet vanskeleg let seg retta opp att ved feil.\n"
"\n"
"Viss du held fram, vil ei ekstra ssh-teneste bli starta på port '%s'.\n"
"Ønskjer du å halda fram?"

#: ../DistUpgrade/DistUpgradeController.py:269
msgid "Starting additional sshd"
msgstr "Startar endå ein sshd"

#: ../DistUpgrade/DistUpgradeController.py:270
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"For å gjera gjenoppretting etter ein systemfeil lettare, vil ein ekstra sshd "
"verta starta på port '%s'. Dersom noko går gale med ssh-en som køyrer, kan "
"du framleis kopla til den nye.\n"

#: ../DistUpgrade/DistUpgradeController.py:278
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Viss du sit bak ein brannmur, må du kanskje opna denne porten mellombels. "
"Sia dette er potensielt farleg, vert det ikkje gjort automatisk. Du kan opna "
"porten med t.d. \n"
"'%s'"

#: ../DistUpgrade/DistUpgradeController.py:349
#: ../DistUpgrade/DistUpgradeController.py:394
msgid "Can not upgrade"
msgstr "Klarte ikkje å oppgradera"

#: ../DistUpgrade/DistUpgradeController.py:350
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Å oppgradera frå «%s» til «%s» er ikkje støtta av dette verktøyet."

#: ../DistUpgrade/DistUpgradeController.py:359
msgid "Sandbox setup failed"
msgstr "Sandkasseoppsett var mislukka"

#: ../DistUpgrade/DistUpgradeController.py:360
msgid "It was not possible to create the sandbox environment."
msgstr "Klarte ikkje å oppretta sandkassemiljøet."

#: ../DistUpgrade/DistUpgradeController.py:366
msgid "Sandbox mode"
msgstr "Sandkassemodus"

#: ../DistUpgrade/DistUpgradeController.py:367
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a systemdir from now until the next reboot are "
"permanent."
msgstr ""
"Denne oppgraderinga køyrer i sandkasse-modus (test-modus). Alle endringar "
"vert skrivne til «%s», og vert fjerna ved neste omstart."

#: ../DistUpgrade/DistUpgradeController.py:395
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"Python-installasjonen din er øydelagd. Den symbolske lenkja "
"«/usr/bin/python» må reparerast."

#: ../DistUpgrade/DistUpgradeController.py:421
msgid "Package 'debsig-verify' is installed"
msgstr "Pakken «debsig-verify» er installert"

#: ../DistUpgrade/DistUpgradeController.py:422
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"Oppgraderinga kan ikkje halda fram med den pakken installert.\n"
"Fjern han med «synaptic» eller «apt-get remove debsig-verify» fyrst og køyr "
"oppdateringa om att."

#: ../DistUpgrade/DistUpgradeController.py:432
msgid "Include latest updates from the Internet?"
msgstr "Inkluder dei siste oppdateringane frå Internett?"

#: ../DistUpgrade/DistUpgradeController.py:433
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Oppgraderingsystemet kan bruka internett til å installera dei nyaste "
"oppdateringane under oppgraderinga. Har du internettsamband er dette sterkt "
"tilrådd.\n"
"\n"
"Oppgraderinga vil ta lenger tid, men systemet vil vera heilt oppdatert når "
"oppgraderinga er ferdig. Du kan velja ikkje å gjera dette, men du bør "
"installera dei siste oppdateringane så fort som mogleg etter oppgraderinga.\n"
"Svarar du «nei» her, vert ikkje nettet brukt i det heile."

#: ../DistUpgrade/DistUpgradeController.py:653
#, python-format
msgid "disabled on upgrade to %s"
msgstr "deaktivert under oppgradering til %s"

#: ../DistUpgrade/DistUpgradeController.py:680
msgid "No valid mirror found"
msgstr "Fann ikkje noko gyldig spegl"

#: ../DistUpgrade/DistUpgradeController.py:681
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"Det vart ikkje funne tenarar for oppgradering i arkivinformasjonen din. "
"Dette kan skje om du køyrer ein intern tenar eller viss tenarlista er "
"utdatert.\n"
"\n"
"Vil du skriva om «sources.list»? Om du vel «Ja» her, så vert alle "
"oppføringane frå «%s» til «%s» oppdaterte. Vel du «Nei», vert oppdateringa "
"avbroten."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:701
msgid "Generate default sources?"
msgstr "Vil du oppretta standardkjelder?"

#: ../DistUpgrade/DistUpgradeController.py:702
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Ingen gyldige oppføringar for «%s» vart funne i «sources.list».\n"
"\n"
"Skal standardoppføringar for «%s» leggjast til? Vel du «Nei», vert "
"oppgraderinga avbroten."

#: ../DistUpgrade/DistUpgradeController.py:737
msgid "Repository information invalid"
msgstr "Ugyldig arkivinformasjon"

#: ../DistUpgrade/DistUpgradeController.py:738
msgid ""
"Upgrading the repository information resulted in a invalid file. Please "
"report this as a bug using the command 'ubuntu-bug update-manager' in a "
"terminal."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:746
msgid "Third party sources disabled"
msgstr "Tredjepartskjelder er deaktiverte"

#: ../DistUpgrade/DistUpgradeController.py:747
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Somme tredjeparts pakkekjelder verkar ikkje lenger. Du kan halda fram med å "
"bruka dei etter oppdateringa gjennom «Eigenskapar for programvare» eller med "
"Synaptic."

#: ../DistUpgrade/DistUpgradeController.py:787
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Pakke i dårleg stand"
msgstr[1] "Pakkar i dårleg stand"

#: ../DistUpgrade/DistUpgradeController.py:790
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Pakken «%s» er ufullstendig og må installerast på nytt, men ingen av dei "
"nødvendige arkiva vart funne. Installer pakken på nytt manuelt, eller fjern "
"han frå systemet."
msgstr[1] ""
"Pakkane «%s» er ufullstendige og må installerast på nytt, men ingen av dei "
"nødvendige arkiva vart funne. Installer pakkane på nytt manuelt, eller fjern "
"dei frå systemet."

#: ../DistUpgrade/DistUpgradeController.py:838
msgid "Error during update"
msgstr "Feil under oppdatering"

#: ../DistUpgrade/DistUpgradeController.py:839
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Eit problem oppstod under oppdateringa. Dette er vanlegvis ei form for "
"nettverksproblem. Sjekk nettverkstilkoplinga di og prøv om att."

#. print "on_button_install_clicked"
#: ../DistUpgrade/DistUpgradeController.py:848
#: ../UpdateManager/UpdateManager.py:747
msgid "Not enough free disk space"
msgstr "Ikkje nok ledig diskplass"

#: ../DistUpgrade/DistUpgradeController.py:849
#, python-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of %s free space on disk "
"'%s'. Please free at least an additional %s of disk space on '%s'. Empty "
"your trash and remove temporary packages of former installations using 'sudo "
"apt-get clean'."
msgstr ""
"Oppgraderinga vart avbroten. Det trengst totalt %s ledig plass på disken "
"«%s». Frigjer minst %s diskplass på «%s». Tøm papirkorga og fjern "
"mellombelse pakkar frå tidlegare installasjonar ved hjelp av «sudo apt-get "
"clean»."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:878
#: ../DistUpgrade/DistUpgradeController.py:1630
msgid "Calculating the changes"
msgstr "Reknar ut endringane"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:910
msgid "Do you want to start the upgrade?"
msgstr "Vil du starta oppdateringa?"

#: ../DistUpgrade/DistUpgradeController.py:976
msgid "Upgrade canceled"
msgstr "Oppgradering avbroten"

#: ../DistUpgrade/DistUpgradeController.py:977
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Oppgraderinga vil no bli avbroten, og det opphavlege systemet vil verta "
"atterreist. Du kan venda tilbake til oppgraderinga når som helst."

#: ../DistUpgrade/DistUpgradeController.py:983
#: ../DistUpgrade/DistUpgradeController.py:1109
msgid "Could not download the upgrades"
msgstr "Klarte ikkje å lasta ned oppgraderingane"

#: ../DistUpgrade/DistUpgradeController.py:984
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1060
#: ../DistUpgrade/DistUpgradeController.py:1097
#: ../DistUpgrade/DistUpgradeController.py:1202
msgid "Error during commit"
msgstr "Feil under innsending"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1062
#: ../DistUpgrade/DistUpgradeController.py:1099
#: ../DistUpgrade/DistUpgradeController.py:1241
msgid "Restoring original system state"
msgstr "Gjenopprettar systemet til sin opphavlege tilstand"

#: ../DistUpgrade/DistUpgradeController.py:1063
#: ../DistUpgrade/DistUpgradeController.py:1078
#: ../DistUpgrade/DistUpgradeController.py:1100
msgid "Could not install the upgrades"
msgstr "Klarte ikkje å installera oppgraderingane"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1068
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Oppgraderinga vart avbroten. Systemet ditt kan vera ubrukeleg. Ei "
"gjenoppretting vert no køyrd (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1073
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug using the command 'ubuntu-bug update-manager' in a "
"terminal and include the files in /var/log/dist-upgrade/ in the bug report.\n"
"%s"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1110
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Oppgraderinga vart avbroten. Kontroller internett-tilkoplinga eller "
"installasjonsmediet og prøv på nytt. "

#: ../DistUpgrade/DistUpgradeController.py:1190
msgid "Remove obsolete packages?"
msgstr "Ønskjer du å fjerna utdaterte pakkar?"

#: ../DistUpgrade/DistUpgradeController.py:1191
#: ../DistUpgrade/DistUpgrade.ui.h:22
msgid "_Keep"
msgstr "_Ta vare på"

#: ../DistUpgrade/DistUpgradeController.py:1191
msgid "_Remove"
msgstr "_Fjern"

#: ../DistUpgrade/DistUpgradeController.py:1203
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Eit problem oppstod under oppryddinga. Sjå meldinga under for meir "
"informasjon. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1279
msgid "Required depends is not installed"
msgstr "Nødvendige avhengnader er ikkje installerte"

#: ../DistUpgrade/DistUpgradeController.py:1280
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Den nødvendige avhengnaden \"%s\" er ikkje installert. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1527
#: ../DistUpgrade/DistUpgradeController.py:1597
msgid "Checking package manager"
msgstr "Sjekkar pakkehandsamaren"

#: ../DistUpgrade/DistUpgradeController.py:1532
msgid "Preparing the upgrade failed"
msgstr "Klarte ikkje å førebu oppgraderinga"

#: ../DistUpgrade/DistUpgradeController.py:1533
msgid ""
"Preparing the system for the upgrade failed. Please report this using the "
"command 'ubuntu-bug update-manager' in a terminal and include the files in "
"/var/log/dist-upgrade/ in the bug report."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1549
msgid "Getting upgrade prerequisites failed"
msgstr "Klarte ikkje å lasta ned oppgraderingskrava"

#: ../DistUpgrade/DistUpgradeController.py:1550
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Please report this as a bug using the command 'ubuntu-bug update-manager' in "
"a terminal and include the files in /var/log/dist-upgrade/ in the bug report."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1581
msgid "Updating repository information"
msgstr "Oppdaterer informasjon om arkivet"

#: ../DistUpgrade/DistUpgradeController.py:1588
msgid "Failed to add the cdrom"
msgstr "Klarte ikkje å leggja til CD-en"

#: ../DistUpgrade/DistUpgradeController.py:1589
msgid "Sorry, adding the cdrom was not successful."
msgstr "Beklagar, å leggja til CD-en var mislukka."

#: ../DistUpgrade/DistUpgradeController.py:1617
msgid "Invalid package information"
msgstr "Ugyldig pakkeinformasjon"

#: ../DistUpgrade/DistUpgradeController.py:1618
#, python-format
msgid ""
"After your package information was updated the essential package '%s' can "
"not be found anymore.\n"
"This indicates a serious error, please report this bug using the command "
"'ubuntu-bug update-manager' in a terminal and include the files in "
"/var/log/dist-upgrade/ in the bug report."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1636
#: ../DistUpgrade/DistUpgradeController.py:1689
msgid "Fetching"
msgstr "Hentar"

#: ../DistUpgrade/DistUpgradeController.py:1642
#: ../DistUpgrade/DistUpgradeController.py:1693
msgid "Upgrading"
msgstr "Oppgraderer"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1647
#: ../DistUpgrade/DistUpgradeController.py:1695
#: ../DistUpgrade/DistUpgradeController.py:1702
#: ../DistUpgrade/DistUpgradeController.py:1713
msgid "Upgrade complete"
msgstr "Oppgradering fullført"

#: ../DistUpgrade/DistUpgradeController.py:1648
#: ../DistUpgrade/DistUpgradeController.py:1696
#: ../DistUpgrade/DistUpgradeController.py:1703
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr "Oppgraderinga er ferdig, men nokre feil dukka opp under prosessen."

#: ../DistUpgrade/DistUpgradeController.py:1655
msgid "Searching for obsolete software"
msgstr "Søkjer etter utdatert programvare"

#: ../DistUpgrade/DistUpgradeController.py:1664
msgid "System upgrade is complete."
msgstr "Systemoppgraderinga er fullført."

#: ../DistUpgrade/DistUpgradeController.py:1714
msgid "The partial upgrade was completed."
msgstr "Delvis oppgradering er ferdig."

#: ../DistUpgrade/DistUpgradeQuirks.py:201
msgid "evms in use"
msgstr "«evms» i bruk"

#: ../DistUpgrade/DistUpgradeQuirks.py:202
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"Systemet ditt brukar «evms» for å handsama dataområde i «/proc/mounts». "
"«evms»-programvaren er ikkje lenger støtta, slå henne av og køyr "
"oppgraderinga på nytt."

#: ../DistUpgrade/DistUpgradeQuirks.py:506
msgid "Your graphics hardware may not be fully supported in Ubuntu 11.04."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:508
msgid ""
"The support in Ubuntu 11.04 for your intel graphics hardware is limited and "
"you may encounter problems after the upgrade. Do you want to continue with "
"the upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:528
#: ../DistUpgrade/DistUpgradeQuirks.py:556
#: ../DistUpgrade/DistUpgradeQuirks.py:583
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Å oppgradera kan redusera skrivebordseffekter, samt ytinga i spel og andre "
"grafikkintensive program."

#: ../DistUpgrade/DistUpgradeQuirks.py:532
#: ../DistUpgrade/DistUpgradeQuirks.py:560
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04 LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Datamaskina brukar NVIDIA «nvidia»-grafikkdrivaren. Ingen tilgjengelege "
"versjonar av denne drivaren verkar med skjermkortet ditt i Ubuntu 10.04 LTS. "
"\n"
"\n"
"Vil du halda fram?"

#: ../DistUpgrade/DistUpgradeQuirks.py:587
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Datamaskina brukar AMD «fglrx»-grafikkdrivaren. Ingen tilgjengelege "
"versjonar av denne drivaren verkar med maskinvaren din i Ubuntu 10.04 LTS."

#: ../DistUpgrade/DistUpgradeQuirks.py:617
msgid "No i686 CPU"
msgstr "Ikkje ein i686-prosessor"

#: ../DistUpgrade/DistUpgradeQuirks.py:618
msgid ""
"Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
"extension. All packages were built with optimizations requiring i686 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Systemet brukar ein i586-prosessor, eller ein prosessor som ikkje har «cmov»-"
"utvidinga. Alle pakkane krev minimum i686-arkitektur. Det er ikkje mogleg å "
"oppgradera systemet ditt til den nyaste Ubuntu-utgjevinga med den noverande "
"maskinvaren."

#: ../DistUpgrade/DistUpgradeQuirks.py:654
msgid "No ARMv6 CPU"
msgstr "Ingen ARMv6-prosessor"

#: ../DistUpgrade/DistUpgradeQuirks.py:655
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Systemet brukar ein ARM-prosessor som er eldre enn ARMv6-arkitekturen. Alle "
"pakkane i karmic krev minimum ARMv6. Det er ikkje mogleg å oppgradera "
"systemet ditt til den nyaste Ubuntu-utgjevinga med den noverande maskinvaren."

#: ../DistUpgrade/DistUpgradeQuirks.py:675
msgid "No init available"
msgstr "Ingen oppstartsprogram (init) er tilgjengelege"

#: ../DistUpgrade/DistUpgradeQuirks.py:676
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Systemet ditt ser ut til å vera i eit virtualisert miljø utan "
"oppstartsprogram (init), til dømes Linux-VServer. Ubuntu 10.04 LTS fungerer "
"ikkje i slike miljø, og krev ei oppdatering av innstillingane i den "
"virtuelle maskina først.\n"
"\n"
"Er du sikker på at du vil halda fram?"

#: ../DistUpgrade/DistUpgradeMain.py:45
msgid "Sandbox upgrade using aufs"
msgstr "Sandkasseoppgradering ved bruk av aufs"

#: ../DistUpgrade/DistUpgradeMain.py:47
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"Bruk den oppgjevne stien til å søkja etter ei CD-plate med pakkar som kan "
"oppgraderast"

#: ../DistUpgrade/DistUpgradeMain.py:53
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Bruk eit grafisk grensesnitt. Tilgjengelege er: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:56
msgid "*DEPRECATED* this option will be ignored"
msgstr ""

#: ../DistUpgrade/DistUpgradeMain.py:59
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""
"Gjennomfør berre ei delevis oppgradering (inga omskriving av sources.list)"

#: ../DistUpgrade/DistUpgradeMain.py:62
msgid "Disable GNU screen support"
msgstr "Skru av GNU-skjermstøtte"

#: ../DistUpgrade/DistUpgradeMain.py:64
msgid "Set datadir"
msgstr "Vel datamappe"

#. print "mediaChange %s %s" % (medium, drive)
#: ../DistUpgrade/DistUpgradeViewGtk.py:114
#: ../DistUpgrade/DistUpgradeViewGtk3.py:117
#: ../DistUpgrade/DistUpgradeViewKDE.py:189
#: ../UpdateManager/DistUpgradeFetcherKDE.py:147
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Sett '%s' i stasjonen '%s'"

#: ../DistUpgrade/DistUpgradeViewGtk.py:135
#: ../DistUpgrade/DistUpgradeViewGtk3.py:138
#: ../DistUpgrade/DistUpgradeViewKDE.py:203
msgid "Fetching is complete"
msgstr "Ferdig med å henta"

#: ../DistUpgrade/DistUpgradeViewGtk.py:146
#: ../DistUpgrade/DistUpgradeViewGtk3.py:149
#: ../DistUpgrade/DistUpgradeViewKDE.py:216
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Hentar fil %li av %li ved %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:149
#: ../DistUpgrade/DistUpgradeViewGtk.py:295
#: ../DistUpgrade/DistUpgradeViewGtk3.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:301
#: ../DistUpgrade/DistUpgradeViewKDE.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:361
#, python-format
msgid "About %s remaining"
msgstr "Omtrent %s igjen"

#: ../DistUpgrade/DistUpgradeViewGtk.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:155
#: ../DistUpgrade/DistUpgradeViewKDE.py:219
#, python-format
msgid "Fetching file %li of %li"
msgstr "Hentar fil %li av %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:183
#: ../DistUpgrade/DistUpgradeViewGtk3.py:186
#: ../DistUpgrade/DistUpgradeViewKDE.py:253
msgid "Applying changes"
msgstr "Utfører endringar"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:208
#: ../DistUpgrade/DistUpgradeViewGtk3.py:211
#: ../DistUpgrade/DistUpgradeViewKDE.py:266
msgid "dependency problems - leaving unconfigured"
msgstr "problem med avhengnader – set ikkje opp pakken"

#: ../DistUpgrade/DistUpgradeViewGtk.py:213
#: ../DistUpgrade/DistUpgradeViewGtk3.py:216
#: ../DistUpgrade/DistUpgradeViewKDE.py:268
#, python-format
msgid "Could not install '%s'"
msgstr "Kunne ikkje installere '%s'"

#: ../DistUpgrade/DistUpgradeViewGtk.py:214
#: ../DistUpgrade/DistUpgradeViewGtk3.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:269
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Oppgraderinga vil halda fram, men pakken «%s» vil kanskje ikkje fungera. "
"Vurder å senda inn ei feilmelding om dette."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:231
#: ../DistUpgrade/DistUpgradeViewGtk3.py:234
#: ../DistUpgrade/DistUpgradeViewKDE.py:290
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Vil du erstatta den egendefinerte oppsettsfila\n"
"\"%s\"?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:232
#: ../DistUpgrade/DistUpgradeViewGtk3.py:235
#: ../DistUpgrade/DistUpgradeViewKDE.py:291
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Du vil mista forandringar du har gjort i denne konfigurasjonsfila om du vel "
"å byte den ut med ein nyare versjon."

#: ../DistUpgrade/DistUpgradeViewGtk.py:250
#: ../DistUpgrade/DistUpgradeViewGtk3.py:253
#: ../DistUpgrade/DistUpgradeViewKDE.py:313
msgid "The 'diff' command was not found"
msgstr "Kommandoen \"diff\" vart ikkje funnen"

#: ../DistUpgrade/DistUpgradeViewGtk.py:463
#: ../DistUpgrade/DistUpgradeViewGtk3.py:469
#: ../DistUpgrade/DistUpgradeViewText.py:90
msgid "A fatal error occurred"
msgstr "Ein kritisk feil oppstod"

#: ../DistUpgrade/DistUpgradeViewGtk.py:464
#: ../DistUpgrade/DistUpgradeViewGtk3.py:470
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Rapporter dette som ein feil (om du ikkje allereie har gjort det), og "
"inkluder filene /var/log/dist-upgrade/main.log og /var/log/dist-"
"upgrade/apt.log i rapporten. Oppgraderinga er avbroten.\n"
"Den opphavlege sources.list er lagra i /etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk.py:481
#: ../DistUpgrade/DistUpgradeViewGtk3.py:487
msgid "Ctrl-c pressed"
msgstr "Ctrl + C trykt"

#: ../DistUpgrade/DistUpgradeViewGtk.py:482
#: ../DistUpgrade/DistUpgradeViewGtk3.py:488
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Dette vil avbryta operasjonen og kanskje setja systemet ditt i ein ubrukeleg "
"tilstand. Er du sikker på at du vil gjera dette?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:630
#: ../DistUpgrade/DistUpgradeViewGtk3.py:621
msgid "To prevent data loss close all open applications and documents."
msgstr "For å hindra tap av data, bør du lukke alle program og dokument."

#: ../DistUpgrade/DistUpgradeViewGtk.py:644
#: ../DistUpgrade/DistUpgradeViewGtk3.py:635
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Ikkje lenger støtta av Canonical (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:645
#: ../DistUpgrade/DistUpgradeViewGtk3.py:636
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Nedgrader (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:646
#: ../DistUpgrade/DistUpgradeViewGtk3.py:637
#, python-format
msgid "Remove (%s)"
msgstr "Fjern (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:647
#: ../DistUpgrade/DistUpgradeViewGtk3.py:638
#, python-format
msgid "No longer needed (%s)"
msgstr "Ikkje lenger nødvendig (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:648
#: ../DistUpgrade/DistUpgradeViewGtk3.py:639
#, python-format
msgid "Install (%s)"
msgstr "Installer (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:649
#: ../DistUpgrade/DistUpgradeViewGtk3.py:640
#, python-format
msgid "Upgrade (%s)"
msgstr "Oppgrader (%s)"

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeViewKDE.py:190
#: ../UpdateManager/DistUpgradeFetcherKDE.py:149
msgid "Media Change"
msgstr "Medieendring"

#: ../DistUpgrade/DistUpgradeViewKDE.py:325
msgid "Show Difference >>>"
msgstr "Vis forskjellar >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:328
msgid "<<< Hide Difference"
msgstr "<<< Skjul forskjellar"

#: ../DistUpgrade/DistUpgradeViewKDE.py:543
msgid "Error"
msgstr "Feil"

#: ../DistUpgrade/DistUpgradeViewKDE.py:555
msgid "&Cancel"
msgstr "&Avbryt"

#: ../DistUpgrade/DistUpgradeViewKDE.py:557
#: ../DistUpgrade/DistUpgradeViewKDE.py:799
msgid "&Close"
msgstr "&Lukk"

#: ../DistUpgrade/DistUpgradeViewKDE.py:603
msgid "Show Terminal >>>"
msgstr "Syn terminal >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:606
msgid "<<< Hide Terminal"
msgstr "<<< Gøym terminal"

#: ../DistUpgrade/DistUpgradeViewKDE.py:686
msgid "Information"
msgstr "Informasjon"

#: ../DistUpgrade/DistUpgradeViewKDE.py:737
#: ../DistUpgrade/DistUpgradeViewKDE.py:782
#: ../DistUpgrade/DistUpgradeViewKDE.py:785 ../DistUpgrade/DistUpgrade.ui.h:11
msgid "Details"
msgstr "Detaljar"

#: ../DistUpgrade/DistUpgradeViewKDE.py:763
#, python-format
msgid "No longer supported %s"
msgstr "Ikkje lenger støtta %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:765
#, python-format
msgid "Remove %s"
msgstr "Fjern %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:767
#: ../DistUpgrade/DistUpgradeViewText.py:179
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Fjern (vart installert automatisk) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:769
#, python-format
msgid "Install %s"
msgstr "Installer %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:771
#, python-format
msgid "Upgrade %s"
msgstr "Oppgrader %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:795
#: ../DistUpgrade/DistUpgradeViewText.py:227
msgid "Restart required"
msgstr "Omstart er nødvendig"

#: ../DistUpgrade/DistUpgradeViewKDE.py:795
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Start systemet på ny for å fullføra oppgraderinga</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:798 ../DistUpgrade/DistUpgrade.ui.h:25
#: ../data/gtkbuilder/UpdateManager.ui.h:32
msgid "_Restart Now"
msgstr "_Start på nytt no"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:816
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Vil du avbryte den køyrande oppgraderinga?</big></big>\n"
"\n"
"Systemet kan verta ubrukeleg viss du avbryt oppgraderinga. Det er sterkt "
"tilrådd å fullføra oppgraderinga."

#: ../DistUpgrade/DistUpgradeViewKDE.py:820
msgid "Cancel Upgrade?"
msgstr "Avbryt oppgradering?"

#: ../DistUpgrade/DistUpgradeView.py:59
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li dag"
msgstr[1] "%li dagar"

#: ../DistUpgrade/DistUpgradeView.py:61
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li time"
msgstr[1] "%li timar"

#: ../DistUpgrade/DistUpgradeView.py:63
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minutt"
msgstr[1] "%li minutt"

#: ../DistUpgrade/DistUpgradeView.py:64
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li sekund"
msgstr[1] "%li sekund"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:80
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:98
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:147
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Nedlastinga vil ta omlag %s med ei DSL-tilkopling på 1 Mbit og omlag %s med "
"eit 56k-modem."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:151
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Denne nedlastinga vil ta ca. %s med tilkoplinga di. "

#: ../DistUpgrade/DistUpgradeView.py:253 ../DistUpgrade/DistUpgrade.ui.h:16
msgid "Preparing to upgrade"
msgstr "Førebur oppgraderinga"

#: ../DistUpgrade/DistUpgradeView.py:254
msgid "Getting new software channels"
msgstr "Hentar nye programvarekanalar"

#: ../DistUpgrade/DistUpgradeView.py:255 ../DistUpgrade/DistUpgrade.ui.h:14
msgid "Getting new packages"
msgstr "Hentar nye programvarepakkar"

#: ../DistUpgrade/DistUpgradeView.py:256 ../DistUpgrade/DistUpgrade.ui.h:15
msgid "Installing the upgrades"
msgstr "Installerer oppgraderingane"

#: ../DistUpgrade/DistUpgradeView.py:257 ../DistUpgrade/DistUpgrade.ui.h:10
msgid "Cleaning up"
msgstr "Ryddar opp"

#: ../DistUpgrade/DistUpgradeView.py:342
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d installert pakke er ikkje lenger støtta av Canonical. Du kan "
"framleis få støtte frå fellesskapet."
msgstr[1] ""
"%(amount)d installerte pakkar er ikkje lenger støtta av Canonical. Du kan "
"framleis få støtte frå fellesskapet."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:351
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d-pakka vil fjernast."
msgstr[1] "%d pakkar vil fjernast."

#: ../DistUpgrade/DistUpgradeView.py:356
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d-pakka vil verta installert."
msgstr[1] "%d pakker vil verta installerte."

#: ../DistUpgrade/DistUpgradeView.py:362
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d-pakka vil verta oppgradert."
msgstr[1] "%d pakkar vil verta oppgraderte."

#: ../DistUpgrade/DistUpgradeView.py:367
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Du må laste ned totalt %s. "

#: ../DistUpgrade/DistUpgradeView.py:371
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be cancelled."
msgstr ""
"Å henta og installera oppgraderinga kan ta fleire timar. Når nedlastinga er "
"ferdig, kan ikkje prosessen verta avbroten."

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:377 ../UpdateManager/UpdateManager.py:664
msgid "Your system is up-to-date"
msgstr "Systemet ditt er oppdatert!"

#: ../DistUpgrade/DistUpgradeView.py:378
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Det er ingen oppgraderingar tilgjengelege for ditt system. Oppgraderinga vil "
"no verta avbroten."

#: ../DistUpgrade/DistUpgradeView.py:391
msgid "Reboot required"
msgstr "Omstart er nødvendig"

#: ../DistUpgrade/DistUpgradeView.py:392
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Oppgraderinga er ikkje fullført og ein omstart er nødvendig. Vil du gjera "
"dette no?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:131
#: ../DistUpgrade/DistUpgradeFetcherCore.py:132
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:131
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:132
msgid "Could not run the upgrade tool"
msgstr "Klarte ikkje å køyra oppgraderingsverktøyet"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:132
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:132
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug update-manager'."
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:207
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:207
msgid "Upgrade tool signature"
msgstr "Signaturen til oppgraderingsvertøyet"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:214
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:214
msgid "Upgrade tool"
msgstr "Oppgraderingsvertøy"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:248
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:248
msgid "Failed to fetch"
msgstr "Klarte ikkje å henta"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:249
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Klarte ikkje å henta oppgraderinga. Det kan vera eit nettverksproblem. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:253
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:253
msgid "Failed to extract"
msgstr "Mislukkast i å pakka ut"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:254
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:254
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Klarte ikkje å pakka ut oppgraderinga. Det kan vera eit problem med "
"netverket eller tenaren. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:259
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:259
msgid "Verification failed"
msgstr "Verifiseringsfeil"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:260
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:260
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Å verifisera oppgraderinga mislukkast. Det kan vera ein feil med nettverket "
"eller tenaren. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:265
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:265
msgid "Authentication failed"
msgstr "Å stadfesta identiteten mislukkast"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:266
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:266
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Klarte ikkje å stadfesta identiteten til oppgraderinga. Det kan vere feil "
"med nettverket eller tenaren. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:280
#: ../DistUpgrade/DistUpgradeFetcherCore.py:286
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:280
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:286
msgid "Can not run the upgrade"
msgstr "Klarte ikkje å køyra oppgradering"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:281
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:281
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:287
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:287
#, python-format
msgid "The error message is '%s'."
msgstr "Feilmeldinga er «%s»."

#: ../DistUpgrade/DistUpgradeViewText.py:91
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Rapporter dette som ein feil og inkluder filene /var/log/dist-"
"upgrade/main.log og /var/log/dist-upgrade/apt.log i rapporten. Oppgraderinga "
"er avbroten. \n"
"Den opphavlege sources.list er lagra i /etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:115
msgid "Aborting"
msgstr "Avbryt"

#: ../DistUpgrade/DistUpgradeViewText.py:120
msgid "Demoted:\n"
msgstr "Degradert:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:127
msgid "To continue please press [ENTER]"
msgstr "Trykk Enter for å halda fram"

#: ../DistUpgrade/DistUpgradeViewText.py:155
#: ../DistUpgrade/DistUpgradeViewText.py:193
#: ../DistUpgrade/DistUpgradeViewText.py:200
msgid "Continue [yN] "
msgstr "Hald fram [jN] "

#: ../DistUpgrade/DistUpgradeViewText.py:155
#: ../DistUpgrade/DistUpgradeViewText.py:193
msgid "Details [d]"
msgstr "Detaljar [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:159
#: ../DistUpgrade/DistUpgradeViewText.py:203
msgid "y"
msgstr "j"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:162
#: ../DistUpgrade/DistUpgradeViewText.py:210
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:170
#, python-format
msgid "No longer supported: %s\n"
msgstr "Ikkje lenger støtta: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:175
#, python-format
msgid "Remove: %s\n"
msgstr "Fjern %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:185
#, python-format
msgid "Install: %s\n"
msgstr "Installer: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:190
#, python-format
msgid "Upgrade: %s\n"
msgstr "Oppgrader %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:207
msgid "Continue [Yn] "
msgstr "Hald fram [Jn] "

#: ../DistUpgrade/DistUpgradeViewText.py:228
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"For å fullføra oppgraderinga, er ein omstart av datamaskina nødvendig.\n"
"Dersom du svarar 'j', vil datamaskinen starta på nytt."

#: ../DistUpgrade/DistUpgrade.ui.h:1
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.ui.h:2
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Vil du avbryta oppgraderinga?</big></b>\n"
"Systemet kan bli ubrukeleg om du avbryt oppgraderinga. Du er oppmoda på det "
"sterkaste å forsetja oppgraderinga."

#: ../DistUpgrade/DistUpgrade.ui.h:5
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Start systemet på nytt for å fullføra oppgraderinga</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:8
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Start oppgraderinga?</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:9
msgid "<b><big>Upgrading Ubuntu to version 11.10</big></b>"
msgstr ""

#: ../DistUpgrade/DistUpgrade.ui.h:12
msgid "Difference between the files"
msgstr "Skilnad mellom filene"

#: ../DistUpgrade/DistUpgrade.ui.h:13
msgid "Distribution Upgrade"
msgstr "Distribusjonsoppgradering"

#: ../DistUpgrade/DistUpgrade.ui.h:17
msgid "Restarting the computer"
msgstr "Startar systemet på nytt"

#: ../DistUpgrade/DistUpgrade.ui.h:18
msgid "Setting new software channels"
msgstr "Endrar kanalene for programvare"

#: ../DistUpgrade/DistUpgrade.ui.h:19
msgid "Terminal"
msgstr "Terminal"

#: ../DistUpgrade/DistUpgrade.ui.h:20
msgid "_Cancel Upgrade"
msgstr "_Avbryt oppgradering"

#: ../DistUpgrade/DistUpgrade.ui.h:21
msgid "_Continue"
msgstr "_Hald fram"

#: ../DistUpgrade/DistUpgrade.ui.h:23
msgid "_Replace"
msgstr "_Erstatt"

#: ../DistUpgrade/DistUpgrade.ui.h:24
msgid "_Report Bug"
msgstr "_Rapporter ein feil"

#: ../DistUpgrade/DistUpgrade.ui.h:26
msgid "_Resume Upgrade"
msgstr "_Forsett oppgraderinga"

#: ../DistUpgrade/DistUpgrade.ui.h:27
msgid "_Start Upgrade"
msgstr "_Start oppgradering"

#: ../UpdateManager/backend/InstallBackendSynaptic.py:55
msgid "Please wait, this can take some time."
msgstr "Vent litt, dette kan ta ei stund."

#: ../UpdateManager/backend/InstallBackendSynaptic.py:57
msgid "Update is complete"
msgstr "Oppdateringa er ferdig"

#: ../UpdateManager/DistUpgradeFetcher.py:108
#: ../UpdateManager/DistUpgradeFetcherKDE.py:102
msgid "Could not find the release notes"
msgstr "Klarte ikkje å finna versjonsnotata"

#: ../UpdateManager/DistUpgradeFetcher.py:109
#: ../UpdateManager/DistUpgradeFetcherKDE.py:103
msgid "The server may be overloaded. "
msgstr "Tenaren kan vera overbelasta. "

#: ../UpdateManager/DistUpgradeFetcher.py:119
#: ../UpdateManager/DistUpgradeFetcherKDE.py:107
msgid "Could not download the release notes"
msgstr "Klarte ikkje å lasta ned versjonsnotata"

#: ../UpdateManager/DistUpgradeFetcher.py:120
#: ../UpdateManager/DistUpgradeFetcherKDE.py:108
msgid "Please check your internet connection."
msgstr "Sjekk internettilkoplinga di."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:61
#: ../UpdateManager/DistUpgradeFetcherKDE.py:84
msgid "Upgrade"
msgstr "Oppgrader"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:88
#: ../data/gtkbuilder/UpdateManager.ui.h:13
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Release Notes"
msgstr "Versjonsnotat"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:127
#: ../UpdateManager/DistUpgradeFetcherKDE.py:140
#: ../UpdateManager/DistUpgradeFetcherKDE.py:142
msgid "Downloading additional package files..."
msgstr "Lastar ned ekstra pakkefiler..."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:140
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Fil %s av %s, %sB/s"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:142
#, python-format
msgid "File %s of %s"
msgstr "Fil %s av %s"

#: ../UpdateManager/ChangelogViewer.py:73
msgid "Open Link in Browser"
msgstr "Opna lenkje i nettlesar"

#: ../UpdateManager/ChangelogViewer.py:76
msgid "Copy Link to Clipboard"
msgstr "Kopier lenkje til utklippstavle"

#: ../UpdateManager/GtkProgress.py:159
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Lastar ned fil %(current)li av %(total)li med %(speed)s/s"

#: ../UpdateManager/GtkProgress.py:164
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Lastar ned fila %(current)li av %(total)li"

#: ../UpdateManager/UpdateManager.py:105 ../do-release-upgrade:90
msgid "Your Ubuntu release is not supported anymore."
msgstr "Di utgåve av Ubuntu er ikkje lenger støtta."

#: ../UpdateManager/UpdateManager.py:106
msgid ""
"You will not get any further security fixes or critical updates. Please "
"upgrade to a later version of Ubuntu Linux."
msgstr ""
"Datamaskina di vil ikkje lenger få kritiske oppdateringar eller "
"tryggleikshol tetta. Oppgrader til ein nyare versjon av Ubuntu Linux no."

#: ../UpdateManager/UpdateManager.py:114
msgid "Upgrade information"
msgstr "Oppgraderingsinformasjon"

#. upload_archive = version_match.group(2).strip()
#: ../UpdateManager/UpdateManager.py:389
#, python-format
msgid "Version %s: \n"
msgstr "Versjon %s: \n"

#: ../UpdateManager/UpdateManager.py:445
msgid ""
"No network connection detected, you can not download changelog information."
msgstr ""
"Inga nettverkstilkopling er oppdaga. Du kan ikkje laste ned informasjon om "
"endringsloggen."

#: ../UpdateManager/UpdateManager.py:453
msgid "Downloading list of changes..."
msgstr "Lastar ned lista over endringar..."

#: ../UpdateManager/UpdateManager.py:497
msgid "_Deselect All"
msgstr ""

#: ../UpdateManager/UpdateManager.py:503
msgid "Select _All"
msgstr ""

#: ../UpdateManager/UpdateManager.py:552
#, python-format
msgid "%(count)s update has been selected."
msgid_plural "%(count)s updates have been selected."
msgstr[0] "%(count)s oppdatering er vald"
msgstr[1] "%(count)s oppdateringar er valde"

#: ../UpdateManager/UpdateManager.py:556
#, python-format
msgid "%s will be downloaded."
msgstr "%s vil verta lasta ned."

#: ../UpdateManager/UpdateManager.py:568
msgid "The update has already been downloaded, but not installed"
msgid_plural "The updates have already been downloaded, but not installed"
msgstr[0] "Oppdateringa er lasta ned, men ikkje installert"
msgstr[1] "Oppdateringane er lasta ned, men ikkje installerte"

#: ../UpdateManager/UpdateManager.py:573
msgid "There are no updates to install"
msgstr "Ingen oppdateringar å installera"

#. TRANSLATORS: this allows to switch the order of the count of
#. updates and the download size string (if needed)
#: ../UpdateManager/UpdateManager.py:579
#, python-format
msgid "%(count_str)s %(download_str)s"
msgstr "%(count_str)s %(download_str)s"

#: ../UpdateManager/UpdateManager.py:586
msgid "Unknown download size."
msgstr "Ukjend storleik på nedlastinga."

#: ../UpdateManager/UpdateManager.py:612
msgid ""
"It is unknown when the package information was updated last. Please click "
"the 'Check' button to update the information."
msgstr ""

#: ../UpdateManager/UpdateManager.py:618
#, python-format
msgid ""
"The package information was last updated %(days_ago)s days ago.\n"
"Press the 'Check' button below to check for new software updates."
msgstr ""
"Pakkeinformasjonen vart sist oppdatert for %(days_ago)s dagar sia\n"
"Trykk «Sjekk»-knappen nedanfor for å leita etter nye oppdateringar."

#: ../UpdateManager/UpdateManager.py:623
#, python-format
msgid "The package information was last updated %(days_ago)s day ago."
msgid_plural ""
"The package information was last updated %(days_ago)s days ago."
msgstr[0] ""
"Pakkeinformasjonen vart sist oppdatert for %(days_ago)s dag sidan."
msgstr[1] ""
"Pakkeinformasjonen vart sist oppdatert for %(days_ago)s dagar sidan."

#: ../UpdateManager/UpdateManager.py:627
#, python-format
msgid "The package information was last updated %(hours_ago)s hour ago."
msgid_plural ""
"The package information was last updated %(hours_ago)s hours ago."
msgstr[0] ""
"Pakkeinformasjonen vart sist oppdatert for %(hours_ago)s time sidan."
msgstr[1] ""
"Pakkeinformasjonen vart sist oppdatert for %(hours_ago)s timar sidan."

#. TRANSLATORS: only in plural form, as %s minutes ago is one of 15, 30, 45 minutes ago
#: ../UpdateManager/UpdateManager.py:632 ../UpdateManager/UpdateManager.py:634
#: ../UpdateManager/UpdateManager.py:636
#, python-format
msgid "The package information was last updated about %s minutes ago."
msgstr ""

#: ../UpdateManager/UpdateManager.py:638
msgid "The package information was just updated."
msgstr ""

#: ../UpdateManager/UpdateManager.py:658
#: ../data/gtkbuilder/UpdateManager.ui.h:22
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr ""
"Programvareoppdateringar fiksar feil, fjernar tryggleikshol og gir deg nye "
"funksjonar."

#: ../UpdateManager/UpdateManager.py:677
msgid "Software updates may be available for your computer."
msgstr "Programvareoppdateringar er kanskje tilgjengelege for maskina di."

#: ../UpdateManager/UpdateManager.py:685
msgid "Welcome to Ubuntu"
msgstr "Velkommen til Ubuntu"

#: ../UpdateManager/UpdateManager.py:687
msgid ""
"These software updates have been issued since this version of Ubuntu was "
"released. If you don't want to install them now, choose \"Update Manager\" "
"from Applications later."
msgstr ""

#: ../UpdateManager/UpdateManager.py:689
msgid ""
"These software updates have been issued since this version of Ubuntu was "
"released. If you don't want to install them now, choose \"Update Manager\" "
"from the Administration Menu later."
msgstr ""

#: ../UpdateManager/UpdateManager.py:692
msgid "Software updates are available for this computer"
msgstr "Programvareoppdateringar er tilgjengelege for denne datamaskina"

#: ../UpdateManager/UpdateManager.py:694
msgid ""
"If you don't want to install them now, choose \"Update Manager\" from "
"Applications later."
msgstr ""
"Om du ikkje vil installera dei no, kan du velja «Oppdateringshandsamar» frå "
"Program-menyen seinare."

#: ../UpdateManager/UpdateManager.py:696
msgid ""
"If you don't want to install them now, choose \"Update Manager\" from the "
"Administration menu later."
msgstr ""
"Om du ikkje vil installera dei no, kan du opna «Oppdateringshandamar» i "
"menyen «Administrasjon» seinare."

#: ../UpdateManager/UpdateManager.py:748
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""
"Oppgraderinga treng %s ledig plass på disken «%s». Frigjer minst %s "
"diskplass på «%s». Tøm papirkorga og fjern mellombelse pakkar frå gamle "
"installasjonar, ved å køyra «sudo apt-get clean»."

#: ../UpdateManager/UpdateManager.py:773
msgid ""
"The computer needs to restart to finish installing updates. Please save your "
"work before continuing."
msgstr ""
"Start datamaskina om att for å fullføra oppdateringane. Hugs å lagra "
"arbeidet ditt før du held fram."

#: ../UpdateManager/UpdateManager.py:837
msgid "Reading package information"
msgstr "Les pakkeinformasjon"

#: ../UpdateManager/UpdateManager.py:852
msgid "Connecting..."
msgstr "Koplar til …"

#: ../UpdateManager/UpdateManager.py:869
msgid "You may not be able to check for updates or download new updates."
msgstr ""
"Det kan henda du ikkje kan sjå etter eller lasta ned nye oppdateringar."

#: ../UpdateManager/UpdateManager.py:994
msgid "Could not initialize the package information"
msgstr "Kunne ikkje førebu pakkeinformasjonen"

#: ../UpdateManager/UpdateManager.py:995
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""
"Eit uløyseleg problem oppstod under klargjering av pakkeinformasjonen.\n"
"\n"
"Rapporter denne feilen i «update-manager»-pakken og inkluder følgjande "
"feilmelding:\n"

#: ../UpdateManager/UpdateManager.py:1024
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""
"Eit uløyseleg problem oppstod medan oppgraderinga vart utrekna.\n"
"\n"
"Rapporter denne feilen i «update-manager»-pakken og inkluder følgjande "
"feilmelding:"

#: ../UpdateManager/UpdateManager.py:1049
msgid " (New install)"
msgstr " (Ny installasjon)"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:1056
#, python-format
msgid "(Size: %s)"
msgstr "(Storleik: %s)"

#: ../UpdateManager/UpdateManager.py:1058
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "Frå versjon %(old_version)s til %(new_version)s"

#: ../UpdateManager/UpdateManager.py:1062
#, python-format
msgid "Version %s"
msgstr "Versjon %s"

#: ../UpdateManager/UpdateManager.py:1093 ../do-release-upgrade:102
msgid "Release upgrade not possible right now"
msgstr "Å oppgradere til ny utgåve er ikkje mogleg akkurat no"

#: ../UpdateManager/UpdateManager.py:1094 ../do-release-upgrade:103
#, c-format, python-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"Klarte ikkje å oppgradera til den nye versjonen akkurat no. Prøv på nytt "
"seinare. Tenaren svarte: «%s»"

#: ../UpdateManager/UpdateManager.py:1096 ../check-new-release-gtk:116
msgid "Downloading the release upgrade tool"
msgstr "Lastar ned oppgraderingverktøyet"

#: ../UpdateManager/UpdateManager.py:1103
#, python-format
msgid "<b>New Ubuntu release '%s' is available</b>"
msgstr "<b>Ubuntu-utgjevinga «%s» er no tilgjengeleg</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:1142
msgid "Software index is broken"
msgstr "Programvareoversikta er øydelagd"

#: ../UpdateManager/UpdateManager.py:1143
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"Det er umogleg å installera eller fjerna programvare. Nytt pakkehandsamaren "
"\"Synaptic\" eller køyr \"sudo apt-get install -f\" i ein terminal for å "
"løysa dette problemet."

#: ../UpdateManager/UnitySupport.py:57
msgid "Check for Updates"
msgstr ""

#: ../UpdateManager/UnitySupport.py:66
msgid "Install All Available Updates"
msgstr ""

#: ../UpdateManagerText/UpdateManagerText.py:31
msgid "Cancel"
msgstr "Avbryt"

#: ../UpdateManagerText/UpdateManagerText.py:32
msgid "Install"
msgstr "Installer"

#: ../UpdateManagerText/UpdateManagerText.py:50
msgid "Building Updates List"
msgstr "Byggjer ei liste over oppdateringar"

#: ../UpdateManagerText/UpdateManagerText.py:53
msgid ""
"\n"
"A normal upgrade can not be calculated, please run: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"\n"
"Ei vanleg oppgradering kan ikkje utreknast, køyr:\n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"Dette kan skuldast:\n"
" * Ei tidlegare oppgradering som ikkje vart fullført\n"
" * Problem med noko av den installerte programvara.\n"
" * Uoffisielle programvarepakkar som ikkje er frå Ubuntu\n"
" * Vanlege endringar i ei Ubuntu-utgåve som endå ikkje er utgjeven"

#: ../UpdateManagerText/UpdateManagerText.py:122
msgid "Downloading changelog"
msgstr "Lastar ned endringslogg"

#: ../UpdateManager/Core/MyCache.py:128
#, python-format
msgid "Other updates (%s)"
msgstr "Andre oppdateringar (%s)"

#: ../UpdateManager/Core/MyCache.py:283
#, python-format
msgid ""
"Changes for the versions:\n"
"Installed version: %s\n"
"Available version: %s\n"
"\n"
msgstr ""

#: ../UpdateManager/Core/MyCache.py:297
msgid "This update does not come from a source that supports changelogs."
msgstr ""

#: ../UpdateManager/Core/MyCache.py:303 ../UpdateManager/Core/MyCache.py:324
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""
"Kunne ikkje lasta ned lista med endringar. \n"
"Kontroller sambandet til Internett."

#: ../UpdateManager/Core/MyCache.py:313
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Lista over endringar har ingen relevante oppføringar.\n"
"\n"
"Bruk http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"fram til endringane vert tilgjengelege, eller prøv på nytt seinare."

#: ../UpdateManager/Core/MyCache.py:318
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Lista med endringar er ikkje tilgjengeleg enno.\n"
"\n"
"Bruk http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"fram til endringane vert tilgjengelege, eller prøv att seinare."

#: ../UpdateManager/Core/UpdateList.py:49
msgid "Failed to detect distribution"
msgstr "Klarte ikkje å oppdaga distribusjonen"

#: ../UpdateManager/Core/UpdateList.py:50
#, python-format
msgid "A error '%s' occurred while checking what system you are using."
msgstr "Støytte på ein feil i undersøkinga av kva system du sit på: «%s»."

#: ../UpdateManager/Core/UpdateList.py:61
msgid "Important security updates"
msgstr "Viktige tryggleiksoppdateringar"

#: ../UpdateManager/Core/UpdateList.py:62
msgid "Recommended updates"
msgstr "Tilrådde oppdateringar"

#: ../UpdateManager/Core/UpdateList.py:63
msgid "Proposed updates"
msgstr "Føreslegne oppdateringar"

#: ../UpdateManager/Core/UpdateList.py:64
msgid "Backports"
msgstr "Backports"

#: ../UpdateManager/Core/UpdateList.py:65
msgid "Distribution updates"
msgstr "Distribusjonsoppdateringar"

#: ../UpdateManager/Core/UpdateList.py:70
msgid "Other updates"
msgstr "Andre oppdateringar"

#: ../data/gtkbuilder/UpdateManager.ui.h:1
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""
"<b><big>Du må sjå etter oppdateringar manuelt</big></b>\n"
"\n"
"Systemet ditt sjekkar ikkje for oppdateringar automatisk. Du kan endra dette "
"i <i>Programvarekelder</i> under <i>Oppdateringar</i>-fanen."

#: ../data/gtkbuilder/UpdateManager.ui.h:4
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr "<big><b>Ikkje alle oppdateringar kan verta installerte</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:5
msgid "<big><b>Running on battery</b></big>"
msgstr "<big><b>Køyrer på batteri</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:6
msgid "<big><b>Starting Update Manager</b></big>"
msgstr "<big><b>Startar Oppdateringshandsamar</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:7
msgid "Changes"
msgstr "Endringar"

#: ../data/gtkbuilder/UpdateManager.ui.h:8
msgid "Chec_k"
msgstr "Sjek_k"

#: ../data/gtkbuilder/UpdateManager.ui.h:9
msgid "Co_ntinue"
msgstr "Hald fram"

#: ../data/gtkbuilder/UpdateManager.ui.h:10
msgid "Description"
msgstr "Skildring"

#: ../data/gtkbuilder/UpdateManager.ui.h:11
msgid "Description of update"
msgstr "Skildring av oppdateringa"

#: ../data/gtkbuilder/UpdateManager.ui.h:12
msgid "It’s safer to connect the computer to AC power before updating."
msgstr "Du lyt kopla datamaskina åt ei straumkjelde før du oppdaterer."

#: ../data/gtkbuilder/UpdateManager.ui.h:14
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"Køyr ei delvis oppgradering for å installera så mange oppdateringar som "
"mogleg.\n"
"\n"
"Årsakene kan vera:\n"
" * Ei tidlegare oppgradering som ikkje vart fullført\n"
" * Problem med som av den installerte programvara\n"
" * Uoffisielle programpakkar som ikkje vert leverte av Ubuntu\n"
" * Normale endringar av ei før-utgåve av ein Ubuntu-versjon"

#: ../data/gtkbuilder/UpdateManager.ui.h:21
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "Show progress of individual files"
msgstr "Syn framdrift for enkeltfiler"

#: ../data/gtkbuilder/UpdateManager.ui.h:23
msgid "Starting Update Manager"
msgstr "Startar oppdateringshandsamaren"

#: ../data/gtkbuilder/UpdateManager.ui.h:24
msgid "U_pgrade"
msgstr "O_ppgrader"

#: ../data/gtkbuilder/UpdateManager.ui.h:25
#: ../data/update-manager.desktop.in.h:3
msgid "Update Manager"
msgstr "Oppdateringshandsamar"

#: ../data/gtkbuilder/UpdateManager.ui.h:26
msgid "You are connected via a wireless modem."
msgstr "Du er kopla til via eit trådlaust modem."

#: ../data/gtkbuilder/UpdateManager.ui.h:27
msgid ""
"You are connected via roaming and may be charged for the data consumed by "
"this update."
msgstr ""
"Du er kopla til via roaming, og kan verta kravd penger for datatrafikken åt "
"denne oppdateringa."

#: ../data/gtkbuilder/UpdateManager.ui.h:28
msgid "Your system is running on battery. Are you sure you want to continue?"
msgstr ""
"Datamaskina di køyrer på batteri. Er du sikker på at du vil halda fram?"

#: ../data/gtkbuilder/UpdateManager.ui.h:29
msgid "_Hide this information in the future"
msgstr "Gøym denne innformasjonen i framtida"

#: ../data/gtkbuilder/UpdateManager.ui.h:30
msgid "_Install Updates"
msgstr "_Installer oppdateringar"

#: ../data/gtkbuilder/UpdateManager.ui.h:31
msgid "_Partial Upgrade"
msgstr "_Delvis oppgradering"

#: ../data/gtkbuilder/UpdateManager.ui.h:33
msgid "_Settings..."
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:34
msgid "_Upgrade"
msgstr "_Oppgrader"

#: ../data/gtkbuilder/UpdateManager.ui.h:35
msgid "updates"
msgstr "oppdateringar"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid ""
"<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr "<b>Ein ny versjon av Ubuntu er tilgjengeleg. Vil du oppgradera?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Ask Me Later"
msgstr "Spør meg seinare"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Don't Upgrade"
msgstr "Ikkje oppgrader"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid "Yes, Upgrade Now"
msgstr "Oppgrader no"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid ""
"You can upgrade at a later time by opening Update Manager and click on "
"\"Upgrade\"."
msgstr ""
"Du kan oppdatera på eit seinare tidspunkt ved å opna oppdateringshandsamaren "
"og trykkja på «Oppgrader»."

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:8
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Du har takka nei til å oppgradera til eit nytt Ubuntu"

#: ../data/update-manager.desktop.in.h:1
msgid "Show and install available updates"
msgstr "Syn og installer tilgjengelege oppdateringar"

#: ../data/update-manager.desktop.in.h:2
msgid "Software Updates"
msgstr "Programvareoppdateringar"

#: ../update-manager:54 ../do-release-upgrade:41
msgid "Show version and exit"
msgstr "Syn versjonen og avslutt"

#: ../update-manager:57
msgid "Directory that contains the data files"
msgstr "Mappe som inneheld datafilene"

#: ../update-manager:60
msgid "Check if a new Ubuntu release is available"
msgstr "Sjekk om ei ny Ubuntu-utgjeving er tilgjengeleg"

#: ../update-manager:63 ../do-release-upgrade:44 ../check-new-release-gtk:176
msgid "Check if upgrading to the latest devel release is possible"
msgstr "Sjekk om ei oppgradering til den siste utviklarversjonen er mogeleg"

#: ../update-manager:67
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr "Oppgrader ved å nytta den sist føreslåtte oppgraderingsprosessen"

#: ../update-manager:74
msgid "Do not focus on map when starting"
msgstr "Ikkje sett fokus på område i oppstarten"

#: ../update-manager:77
msgid "Try to run a dist-upgrade"
msgstr "Prøv å køyra ein dist-upgrade"

#: ../update-manager:80
msgid "Do not check for updates when starting"
msgstr "Ikkje leit etter oppdateringar i oppstarten"

#: ../update-manager:84 ../do-release-upgrade:60
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "Test oppgraderinga i ei aufs-sandkasse"

#: ../update-manager:104
msgid "Running partial upgrade"
msgstr "Køyrer delvis oppgradering"

#: ../do-release-upgrade:48 ../check-new-release-gtk:180
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Forsøk å oppgradera til den nyaste utgjevinga ved å bruka "
"oppgraderingsverktøyet frå $distro-proposed"

#: ../do-release-upgrade:52
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Køyr i ein spesiell oppgraderingsmodus.\n"
"No vert 'desktop' støtta for oppgradering av vanlege arbeidsstasjonar, og "
"'server' for tenarsystem."

#: ../do-release-upgrade:58
msgid "Run the specified frontend"
msgstr "Køyr vald grenseflate"

#: ../do-release-upgrade:63
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Berre sjekk om ei ny distribusjonsutgjeving er tilgjengeleg, og returner "
"resultatet via avsluttingskoden."

#: ../do-release-upgrade:77
msgid "Checking for a new ubuntu release"
msgstr "Ser etter ei ny utgjeving av Ubuntu"

#: ../do-release-upgrade:91
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"For informasjon om oppgradering, vitj:\n"
"%(url)s\n"

#: ../do-release-upgrade:97
msgid "No new release found"
msgstr "Fann ingen nye utgjevingar"

#: ../do-release-upgrade:109
#, c-format
msgid "New release '%s' available."
msgstr "Ei ny utgjeving, «%s», er tilgjengeleg."

#: ../do-release-upgrade:110
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Køyr «do-release-upgrade» for å oppgradera til henne."

#: ../check-new-release-gtk:91
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Oppgradering tilgjengeleg: Ubuntu %(version)s"

#: ../check-new-release-gtk:132
#, c-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Du har takka nei til å oppgradera åt Ubuntu %s"

#: ../Janitor/computerjanitor/exc.py:30
#, python-format
msgid "Unimplemented method: %s"
msgstr "Ikkje-innarbeidd metode: %s"

#: ../Janitor/computerjanitor/file_cruft.py:46
msgid "A file on disk"
msgstr "Ei fil på disken"

#: ../Janitor/computerjanitor/package_cruft.py:45
msgid ".deb package"
msgstr ".deb-pakke"

#: ../Janitor/computerjanitor/missing_package_cruft.py:33
msgid "Install missing package."
msgstr "Installer manglande pakke."

#: ../Janitor/computerjanitor/missing_package_cruft.py:42
#, python-format
msgid "Package %s should be installed."
msgstr "Pakka %s lyt verta installert."

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:36
#, python-format
msgid "%i obsolete entries in the status file"
msgstr "%i gamle oppføringar i statusfila"

# Dette er ei fil i /var/lib/dpkg
#: ../Janitor/plugins/dpkg_status_plugin.py:39
msgid "Obsolete entries in dpkg status"
msgstr "Gamle oppføringar i dpkg-status-fila"

# Dette er ei fil i /var/lib/dpkg
#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:42
msgid "Obsolete dpkg status entries"
msgstr "Gamle dpkg-status-oppføringar"

#: ../Janitor/plugins/kdelibs4to5_plugin.py:39
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug #279621 for details."
msgstr ""
"kdelibs5-dev må installerast om du oppgraderer med kdelibs4-dev installert. "
"Sjå bugs.launchpad.net, feilrapport #279621 for detaljar."

#: ../Janitor/plugins/langpack_manual_plugin.py:34
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr "%s må vera merkt som manuelt installert."

#: ../Janitor/plugins/remove_lilo_plugin.py:28
msgid ""
"Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr ""
"Fjern lilo sia grub `og er installert. (Sjå feilrapport #314004 for "
"detaljar.)"

#~ msgid "Show details of an update"
#~ msgstr "Syn detaljar for ei opdatering"

#~ msgid "*deprecated* Check for new distribution releases"
#~ msgstr "Sjå etter nye utgjevingar av distribusjonen"

#~ msgid "_Check All"
#~ msgstr "Marker _alle"

#~ msgid "_Uncheck All"
#~ msgstr "_Fjern all markering"

#~ msgid ""
#~ "This is most likely a bug in the upgrade tool. Please report it as a bug"
#~ msgstr ""
#~ "Dette er mest sannsynleg ein feil i oppgraderingsverktøyet. Ver god og "
#~ "rapporter denne feilen"

#~ msgid ""
#~ "Upgrading the repository information resulted in a invalid file. Please "
#~ "report this as a bug."
#~ msgstr ""
#~ "Oppgraderinga av arkivinformasjonen resulterte i ei ugyldig fil. Rapporter "
#~ "dette som ein feil."

#~ msgid ""
#~ "It was impossible to install a required package. Please report this as a "
#~ "bug. "
#~ msgstr ""
#~ "Ein naudsynt pakke kunne ikkje installerast. Ver venleg og rapporter denne "
#~ "feilen. "

#~ msgid "Previous selected"
#~ msgstr "Tidlegare valde"

#~ msgid "First run welcome message"
#~ msgstr "Fyrstegongs velkomstmelding"

#~ msgid "Auto close the install window after successful install"
#~ msgstr "Lukk vindauget når installeringa er ferdig"

#~ msgid "This change is not coming from a source that supports changelogs."
#~ msgstr "Denne endringa kjem frå ei kjelde som ikkje støttar endringsloggar."

#~ msgid ""
#~ "If none of this applies, then please report this bug against the 'update-"
#~ "manager' package and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report."
#~ msgstr ""
#~ "Viss ingen av desse stemmer, send denne feilrapporten for 'update-manager' "
#~ "og legg ved filene i mappa /var/log/dist-upgrade/ i feilmeldingsrapporten."

#~ msgid ""
#~ "The upgrade has aborted. Please check your Internet connection or "
#~ "installation media and try again. All files downloaded so far are kept."
#~ msgstr ""
#~ "Oppgraderinga vart avbroten. Kontroller tilkoplinga til internett eller "
#~ "installasjonsmediet og prøv på nytt. Filene som er lasta ned så langt vert "
#~ "tekne vare på."

#, python-format
#~ msgid ""
#~ "\n"
#~ "\n"
#~ "Please report this bug against the 'update-manager' package and include the "
#~ "files in /var/log/dist-upgrade/ in the bug report.\n"
#~ "%s"
#~ msgstr ""
#~ "\n"
#~ "\n"
#~ "Ver venleg å sende denne feilrapporten til «update-manager», og legg ved "
#~ "filene i katalogen /var/log/dist-upgrade/ i feilmeldingsrapporten.\n"
#~ "%s"

#~ msgid ""
#~ "Preparing the system for the upgrade failed. Please report this as a bug "
#~ "against the 'update-manager' package and include the files in /var/log/dist-"
#~ "upgrade/ in the bug report."
#~ msgstr ""
#~ "Klarte ikkje å førebu systemet for oppgraderinga. Rapporter dette som ein "
#~ "feil i «update-manager»-pakken og legg ved filene i /var/log/dist-upgrade/ i "
#~ "feilrapporten."

#~ msgid ""
#~ "The system was unable to get the prerequisites for the upgrade. The upgrade "
#~ "will abort now and restore the original system state.\n"
#~ "\n"
#~ "Please report this as a bug against the 'update-manager' package and include "
#~ "the files in /var/log/dist-upgrade/ in the bug report."
#~ msgstr ""
#~ "Systemet klarte ikkje å få tak i dei nødvendige filene for oppgraderinga. Ho "
#~ "vert no avbroten, og den opphavlege tilstanden til systemet vert "
#~ "gjenoppretta. \n"
#~ "\n"
#~ "Rapporter dette som ein feil i «update-manager»-pakken og inkluder filene i "
#~ "/var/log/dist-upgrade/ i feilrapporten."

#, python-format
#~ msgid ""
#~ "After your package information was updated the essential package '%s' can "
#~ "not be found anymore.\n"
#~ "This indicates a serious error, please report this bug against the 'update-"
#~ "manager' package and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report."
#~ msgstr ""
#~ "Etter at pakkeinformasjonen vart oppdatert, finst ikkje pakken «%s» lenger.\n"
#~ "Dette tyder på ein alvorleg feil. Rapporter denne feilen i «update-manager»-"
#~ "pakken og inkluder filene i /var/log/dist-upgrade/ i feilrapporten."

#~ msgid "*DEPRECATED* this option will be ignore"
#~ msgstr "*UTGÅTT* dette valet blir oversett"

#~ msgid ""
#~ "This usually is caused by a system were /tmp is mounted noexec. Please "
#~ "remount without noexec and run the upgrade again."
#~ msgstr ""
#~ "Dette vert som regel skulda at mappa /tmp er montert utan løyve til å køyra "
#~ "program. Monter mappa på nytt med løyve til å køyra program, og køyr "
#~ "oppgraderinga på nytt."

#~ msgid "<b><big>Upgrading Ubuntu to version 11.04</big></b>"
#~ msgstr "<b><big>Oppgraderer Ubuntu til versjon 11.04</big></b>"

#~ msgid ""
#~ "It is unknown when the package information was updated last. Please try "
#~ "clicking on the 'Check' button to update the information."
#~ msgstr ""
#~ "Det er ukjend når pakkeinformasjonen vart oppdatert sist. Prøv å trykkja på "
#~ "\"Sjekk\"-knappen for å oppdatera informasjonen."

#~ msgid "The package information was last updated less than one hour ago."
#~ msgstr ""
#~ "Pakkeinformasjonen vart sist oppdatert for mindre enn ein time sidan."

#~ msgid ""
#~ "These software updates have been issued since Ubuntu was released. If you "
#~ "don't want to install them now, choose \"Update Manager\" from the "
#~ "Administration Menu later."
#~ msgstr ""
#~ "Desse oppdateringane vart utgjevne etter at Ubuntu vart sleppt. Om du ikkje "
#~ "vil installera dei no, kan du opna «Oppdateringshandsamar» i menyen "
#~ "«Administrasjon» seinare."

#, python-format
#~ msgid ""
#~ "Changes for the versions:\n"
#~ "%s\n"
#~ "%s\n"
#~ "\n"
#~ msgstr ""
#~ "Endra mellom versjonane:\n"
#~ "%s\n"
#~ "%s\n"
#~ "\n"

#~ msgid "Settings..."
#~ msgstr "Innstillingar ..."

#~ msgid ""
#~ "If automatic checking for updates is disabled, you have to reload the "
#~ "channel list manually. This option allows to hide the reminder shown in this "
#~ "case."
#~ msgstr ""
#~ "Dersom automatisk søk etter oppdateringar er deaktivert, må du henta lista "
#~ "over pakkekjelder manuelt. Dette valet lar deg gøyma påminninga som vert "
#~ "vist."

#~ msgid "If this key is set a first run welcome message will be presented."
#~ msgstr ""
#~ "Viss denne nøkkelen er vald, vil ei velkomstmelding verta vist ved første "
#~ "oppstart."

#~ msgid ""
#~ "If this key is set the install window will be automatically closed on "
#~ "successful installation."
#~ msgstr ""
#~ "Viss denne nøkkelen er vald, vil installajonsvindauge verta lukka etter ein "
#~ "vellukka installasjon."

#~ msgid ""
#~ "If this key is set the main update list window will show version information "
#~ "(from version to version)."
#~ msgstr ""
#~ "Viss denne nøkkelen er vald, vil hovudvindauget som listar opp oppdateringar "
#~ "syna versjonsinformasjon (frå versjon til versjon)."

#~ msgid ""
#~ "If this key is set, the main update list window will show the package "
#~ "summary before the package name."
#~ msgstr ""
#~ "Viss denne nøkkelen er vald, vil hovudvindauget som listar opp oppdateringar "
#~ "syna oppsummeringa av pakkane framfor namnet."

#~ msgid "Remind to reload the channel list"
#~ msgstr "Minn meg på å lasta kanallista på nytt"

#~ msgid "Show package summary before package name in update list"
#~ msgstr "Syn oppsummeringa av pakkane før namnet i oppdateringslista"

#~ msgid "Show version in update list"
#~ msgstr "Syn versjon i oppdateringslista"

#~ msgid "Stores the size of the update-manager dialog"
#~ msgstr "Lagrar storleiken på dialogvindauget åt oppdateringshandsamaren"

#~ msgid ""
#~ "Stores the state of the expander that contains the list of changes and the "
#~ "description"
#~ msgstr ""
#~ "Lagrar tilstanden til utvidaren, som inneheld ei liste over endringar og "
#~ "skildringa deira"

#~ msgid "The window size"
#~ msgstr "Vindaugsstorleiken"

#~ msgid ""
#~ "This key is deprecated in favor of the file /etc/update-manager/release-"
#~ "upgrades Check automatically if a new version of the current distribution is "
#~ "available and offer to upgrade (if possible)."
#~ msgstr ""
#~ "Denne nøkkelen er utgått til fordel for fila /etc/update-manager/release-"
#~ "upgrades Sjå automatisk etter nye utgjevingar av distribusjonen, og tilby å "
#~ "oppgradera, om mogleg."

#~ msgid ""
#~ "This will permanently hide the new release prompt from check-new-release-"
#~ "gtk. Note that the small button in the main update-manager UI will still be "
#~ "there."
#~ msgstr ""
#~ "Dette vil permanent gøyma utgjevingsvarselet frå check-new-release-gtk. Merk "
#~ "at den lisle knappen i oppdateringshandsamaren framleis vil vera der."

#~ msgid "make check-new-release-gtk ignore a given new release"
#~ msgstr "få check-new-release-gtk til å ignorera ei gjeven ny utgjeving"

#~ msgid ""
#~ "These software updates have been issued since Ubuntu was released. If you "
#~ "don't want to install them now, choose \"Update Manager\" from Applications "
#~ "later."
#~ msgstr ""
#~ "Desse oppdateringane har vorte tilgjengelege etter at Ubuntu vart utgjeve. "
#~ "Om du ikkje vil installera dei no, kan du velja «Oppdateringshandsamar» frå "
#~ "Program-menyen seinare."

#~ msgid "0 KB"
#~ msgstr "0 KiB"

#, python-format
#~ msgid "%.0f KB"
#~ msgstr "%.0f KiB"

#~ msgid "1 KB"
#~ msgstr "1KiB"
